### PR TITLE
WIP: cluster creation

### PR DESCRIPTION
**Pull request by GrigoriyMikhalkin**

### CODE REVIEW

**Review by Klaven — APPROVED** [review]
/lgtm
looks like a good start.

File: main.go [hunk 1]
```diff
@@ -18,57 +18,60 @@ package main
...
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	infrastructurev1alpha1 "github.com/null-channel/cluster-api-provider-civo/api/v1alpha1"
+	infrastructurev1beta1 "github.com/null-channel/cluster-api-provider-civo/api/v1beta1"
+	"github.com/null-channel/cluster-api-provider-civo/controllers"
 	//+kubebuilder:scaffold:imports
 )
 
+const (
+	defaultRegion = "London 1"
```
Comment (s1ntaxe770r) [inline main.go]: regions should use short codes so "London 1" should be "LON1"

Comment (GrigoriyMikhalkin) [reply main.go]: Fixed

Comment (Klaven) [reply main.go]: nice catch!

**Review by s1ntaxe770r — APPROVED** [review]

**Pull request merged**